gtest/src/manager.rs

```diff
@@ -304,6 +305,32 @@ impl ExtManager {
         }
     }
 
+    pub(crate) fn is_user(&self, id: &ProgramId) -> bool {
+        !self.actors.contains_key(id) || matches!(self.actors.get(id), Some((Actor::User, _)))
+    }
+
+    pub(crate) fn add_value_to(&mut self, id: &ProgramId, value: Balance) {
+        let (_, balance) = self.actors.entry(*id).or_insert((Actor::User, 0));
+        *balance = balance.saturating_add(value);
+    }
+
+    pub(crate) fn actor_balance(&self, id: &ProgramId) -> Option<Balance> {
+        self.actors.get(id).map(|(_, balance)| *balance)
+    }
```

Comment:
```suggestion
    pub(crate) fn balance_of(&self, id: &ProgramId) -> Balance {
        self.actors.get(id).map(|(_, &balance)| balance).unwrap_or_default()
    }
```

This seems more logic and canonical for me